src/libraries/System.Security.AccessControl/src/System/Security/AccessControl/ACL.cs

```diff
@@ -68,7 +68,7 @@ object IEnumerator.Current
 
         public GenericAce Current
         {
-            get { return ((IEnumerator)this).Current as GenericAce; }
+            get { return (((IEnumerator)this).Current as GenericAce)!; }
```

Comment:
Why did you use ! here, but a hard cast on GetEnumerator?  Seems like changing as-cast to hard-cast would make this look nicer.

Reply:
Well this is already has a cast and adding another cast might look confusing, but am ok with hard cast